src/main/webRequest/webRequestHandler.test.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {WebRequestHandler} from './webRequestHandler';
+
+jest.mock('electron-log', () => ({
+    silly: jest.fn(),
+}));
+
+describe('main/webRequest/webRequestHandler', () => {
```

Comment:
(nitpick/non-blocking) Nice tests! Since the details are passed to the listeners would it make sense for the mock implementations to have a signature that uses details (e.g. `(details) => result += details.some + '1, 2 or 3'`?

Reply:
The handler class itself doesn't make use of the `details`, so it's a bit irrelevant there actually. It's mostly just a passthrough, so I test it in the manager instead. Bit of a break of the unit test paradigm, but at least this way all of the tests have a bit more meaning.